bin/sao

```diff
@@ -116,18 +118,23 @@ defaultCommand
     type: 'boolean'
   })
   .option('templates', {
+    alias: 'l',
```

Comment:
no need to add a short flag imo

Reply:
So, maybe `ls` flag here?

Reply:
`--templates` just sound more meaningful to me 😅

Reply:
With all due respect, please take a look at (Ubiquitous Language)[https://martinfowler.com/bliki/UbiquitousLanguage.html]

Reply:
Anyway, `--templates` is enough clear... :fist_raised: